tests-js/EscrowReward/state.test.js

```diff
@@ -130,8 +127,8 @@ async function checkStatus(
  * @return [SOV Balance, Reward Token Balance].
  */
 async function getTokenBalances(addr, sovContract, lockedSOVContract) {
-	let sovBal = await sovContract.balanceOf(addr);
-	let rewardBal = await lockedSOVContract.getLockedBalance(addr);
+	let sovBal = (await sovContract.balanceOf(addr)).toNumber();
+	let rewardBal = (await lockedSOVContract.getLockedBalance(addr)).toNumber();
```

Comment:
Sometimes, using toNumber() is not really accurate if the number that is being converted is too big, e.g 100e18 (depends on the processor), so I suggest converting it to string (toString()) which I believe would be more accurate.

Reply:
Solved in commit 1296cfc